common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/BrokerSilentTokenCommandParameters.java

```diff
@@ -89,7 +89,7 @@ public void validate() throws ArgumentException {
                     "mCallerPackageName", "Caller package name is not set"
             );
         }
-        if (SdkType.MSAL == getSdkType() &&
+        if ((SdkType.MSAL == getSdkType() || SdkType.MSALCPP == getSdkType()) &&
```

Comment:
The sdktype is hardcoded in the logic when creating the requests, so technically we should only encounter one of these values.

Reply:
Sure.  Now.  But that might not be true forever.

Reply:
For now, this is an `enum` so I don't expect it to be anything other than the values currently declared in the `SdkType.java`. @AdamBJohnsonx not clear to me how this could be an unknow value....are you anticipating something specific?